theano/tensor/tests/test_basic.py

```diff
@@ -1629,7 +1658,7 @@ def test_py_c_match():
     uint8=[numpy.arange(1, 256, dtype='uint8')],
     complex=(randc128_ranged(1, 1000, (2, 3)),),
     empty=(numpy.asarray([], dtype=config.floatX),),)
-_grad_broadcast_unary_arccosh = dict(normal=(rand_ranged(1, 1000, (2, 3)),),)
+_grad_broadcast_unary_arccosh = dict(normal=(rand_ranged(1 + _eps, 1000, (2, 3)),),)
```

Comment:
Actually, if the value sampled was too close to 1, `verify_grad` would fail. It may happen rarely in practice, but it would eventually.